tools/topology/topology2/include/components/src_format_s32_convert_from_48k.conf

```diff
@@ -27,7 +27,7 @@
 					]
 					out_bit_depth [ 32 ]
 					out_valid_bit_depth [ 32 ]
-					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
+					obs "$[$out_channels * (($out_rate / 1000) + 1) * ($out_bit_depth / 8)]"
```

Comment:
I think this would benefit from more explanation and examples with some values. So the base class version of obs calculation is not enough (output_audio_format.conf), but why is 1 extra word enough in this case. I think a similar documentation note as in output_audio_format.conf would be in order here.

Reply:
I double checked with Seppo, current +1 is for ASRC, somehow mixed with ASRC and SRC, for SRC, the calculation is quite complex, SRC converter is composed with two set of polyphase filter, it is 21/20 and 7/8, which means 20 --> 21, the second is 8 --> samples, Seppo helped generated below table to get better understanding for the max obs:
![image](https://github.com/thesofproject/sof/assets/17277136/9a8153b8-f9c9-470f-a63b-48881879eaff)

we should set obs as 56 for LL pipe, for other cases, we need more parameter for set, like deep buffer.
Let's keep implementation for LL for now

Reply:
@btian1 Have you been able to run e.g. the SRC capture and playback in sof-nocodec topology with clean audio with this patched topology with any of these impacted rates?

Reply:
Can't listen audio in real time with nocodec, but the topology is arranged such that you can capture from PCM the playback to PCM via SSP loopback. Launch arecord from one shell window and aplay from other. Then copy the recording to you PC and listen if it's clean and with correct rate/duration.